### ISSUE TITLE
Different Outputs for Same Input Image

### ISSUE THREAD

**Issue by ParitoshParmar** [BVLC/caffe#4624]
I am trying to do regression. For that I am using AlexNet as base architecture, to which I add an additional FC layer and replace the Softmax layer with Euclidean loss layer. The labels are float values (modified the image data accordingly) scaled in the range of 0 to 1. 

During testing I am getting different output values from the last layer (FC9) for same the input image. Any idea why this might be happening ?

(During training, the loss starts from around 0.002 - which seems quite small)

**Comment by shelhamer** [comment]
During training a net with dropout layers will have different outputs for the same input since the dropout noise will differ each time.

Please ask usage questions on the the mailing list.

From https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md:

> _Please do not post usage, installation, or modeling questions, or other requests for help to Issues._
Use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe.
